### PR TITLE
feat: add docs on how to use the terminal with your shell

### PR DESCRIPTION
**Pull request by TheMeaningfulEngineer**
Ticket: None
Changelog: None

### CODE REVIEW

File: 09.Add-ons/50.Remote-Terminal/docs.md [hunk 1]
```diff
@@ -85,6 +85,25 @@ the terminal output which users can later inspect from the
 !!!!! See [the Mender features page](https://mender.io/product/features?target=_blank)
 !!!!! for an overview of all Mender plans and features.
 
+## Using your own terminal
+
+It is possible to use the remote terminal feature from you own terminal instead of the UI.
+To achieve this please download the [mender-cli](../../10.Downloads/docs.md#mender-cli) tool.
+
+You will need the Device ID and the JWT token, both accessible from the UI.
```
Comment (tranchitella) [inline 09.Add-ons/50.Remote-Terminal/docs.md:93]:
```suggestion
You will need the Device ID and your session token, both accessible from the UI.
```

File: 09.Add-ons/50.Remote-Terminal/docs.md [hunk 2]
```diff
@@ -85,6 +85,25 @@ the terminal output which users can later inspect from the
...
+Device ID is available under the UI under `DEVICES -> select device -> Identity -> ID`.
+For the JWT token you can use whatever way works best for you, a [PAT](../../08.Server-integration/01.Using-the-apis/docs.md#personal-access-tokens) or
```
Comment (tranchitella) [inline 09.Add-ons/50.Remote-Terminal/docs.md:95]:
```suggestion
For the session token you can use whatever way works best for you, a [PAT](../../08.Server-integration/01.Using-the-apis/docs.md#personal-access-tokens) or 
```

File: 09.Add-ons/50.Remote-Terminal/docs.md [hunk 3]
```diff
@@ -85,6 +85,25 @@ the terminal output which users can later inspect from the
...
+
+``` bash
+DEVICE_ID=<Fill with yours>
+SESSION_TOKEN=<Fill with yours>
+
+./mender-cli --server https://eu.hosted.mender.io --token-value $SESSION_TOKEN terminal $DEVICE_ID
+```
+
+
+
```
Comment (tranchitella) [inline 09.Add-ons/50.Remote-Terminal/docs.md:106]: You don't need three new lines here (nit-pick).

File: 09.Add-ons/50.Remote-Terminal/docs.md [hunk 4]
```diff
@@ -85,6 +85,25 @@ the terminal output which users can later inspect from the
...
+You will need the Device ID and the JWT token, both accessible from the UI.
+Device ID is available under the UI under `DEVICES -> select device -> Identity -> ID`.
+For the JWT token you can use whatever way works best for you, a [PAT](../../08.Server-integration/01.Using-the-apis/docs.md#personal-access-tokens) or 
+a session token (`Settings -> My profile -> Session token`) are the quickest.
+
+``` bash
+DEVICE_ID=<Fill with yours>
+SESSION_TOKEN=<Fill with yours>
+
```
Comment (tranchitella) [inline 09.Add-ons/50.Remote-Terminal/docs.md:101]: Actually, if you log in using the mender-cli you don't need to specify the token.
`mender-cli login` and then you can connect to the remote terminal, that's it.

File: 09.Add-ons/50.Remote-Terminal/docs.md (hunk 3 above)
Comment (TheMeaningfulEngineer) [reply 09.Add-ons/50.Remote-Terminal/docs.md:106]: Fixing

File: 09.Add-ons/50.Remote-Terminal/docs.md (hunk 4 above)
Comment (TheMeaningfulEngineer) [reply 09.Add-ons/50.Remote-Terminal/docs.md:101]: Will add this, tnx

File: 09.Add-ons/50.Remote-Terminal/docs.md (hunk 1 above)
Comment (TheMeaningfulEngineer) [reply 09.Add-ons/50.Remote-Terminal/docs.md:93]: I'm suggesting a PAT and a session token as means to achieve this.
Would you like me to remove the PAT and keep in session, or you missed the part where both are used?

File: 09.Add-ons/50.Remote-Terminal/docs.md (hunk 2 above)
Comment (TheMeaningfulEngineer) [reply 09.Add-ons/50.Remote-Terminal/docs.md:96]: Same as
https://github.com/mendersoftware/mender-docs/pull/2238/files#r1360402989

File: 09.Add-ons/50.Remote-Terminal/docs.md (hunk 1 above)
Comment (tranchitella) [reply 09.Add-ons/50.Remote-Terminal/docs.md:93]: The fact that the session token is a JWT is a technical detail; from a user's point of view, it's a token.
I'd suggest removing the word JWT.

**Review by tranchitella — APPROVED** [review]
I pre-approve, use your judgement.
I'd personally avoid the word "JWT" and use "token", "access token" or "session token".

File: 09.Add-ons/50.Remote-Terminal/docs.md (hunk 1 above)
Comment (TheMeaningfulEngineer) [reply 09.Add-ons/50.Remote-Terminal/docs.md]: Deal, using "access token" instead - to make a distinction between session token and PAT

**Pull request merged**